sierra_adapter/sierra_items_to_dynamo/src/test/scala/uk/ac/wellcome/platform/sierra_items_to_dynamo/ServerTest.scala

```diff
@@ -1,34 +1,33 @@
 package uk.ac.wellcome.platform.sierra_items_to_dynamo
 
+import com.gu.scanamo.DynamoFormat
 import com.twitter.finagle.http.Status._
-import com.twitter.finatra.http.EmbeddedHttpServer
-import com.twitter.inject.server.FeatureTest
-import uk.ac.wellcome.test.utils.{
-  AmazonCloudWatchFlag,
-  DynamoDBLocalClients,
-  SQSLocal
-}
+import org.scalatest.FunSpec
+import uk.ac.wellcome.models.transformable.sierra.SierraItemRecord
+import uk.ac.wellcome.test.fixtures.{LocalDynamoDb, SqsFixtures}
 
 class ServerTest
-    extends FeatureTest
-    with AmazonCloudWatchFlag
-    with DynamoDBLocalClients
-    with SQSLocal {
+    extends FunSpec
+    with LocalDynamoDb[SierraItemRecord]
+    with SqsFixtures
+    with fixtures.Server {
 
-  val queueUrl = createQueueAndReturnUrl("sierra-item-to-dynamo-servertest-q")
+  override lazy val evidence = DynamoFormat[SierraItemRecord]
 
-  val server = new EmbeddedHttpServer(
-    new Server(),
-    flags = Map(
-      "aws.dynamo.tableName" -> "serverTestTable",
-      "aws.sqs.queue.url" -> queueUrl
-    ) ++ sqsLocalFlags ++ cloudWatchLocalEndpointFlag ++ dynamoDbLocalEndpointFlags
-  )
+  it("shows the healthcheck message") {
+    withLocalDynamoDbTable { tableName =>
+      withLocalSqsQueue { queueUrl =>
+        val flags = sqsLocalFlags(queueUrl) ++ dynamoDbLocalEndpointFlags ++ Map(
```

Comment:
We should (in a separate patch) modify `dynamoDbLocalEndpointFlags` to:

* Use a consistent name with the other flags `dynamoDbLocalFlags`
* Take the table name as a parameter